src/fragments/lib/analytics/flutter/getting-started/10_preReq.mdx

```diff
@@ -3,7 +3,3 @@
   * An iOS configuration targeting at least iOS 11.0
```

Comment:
I think that could be confusing since analytics isn't implemented for dev preview yet. If customers want to use analytics alongside auth dev preview they'd need to satisfy the minimums for that package, so I don't think it is necessary to address it here.